frontend/ember-cli-build.js

```diff
@@ -4,6 +4,7 @@ const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
 module.exports = function(defaults) {
   let app = new EmberApp(defaults, {
+    hinting: false,
```

Comment:
Is this just disabling template linting? How do we know `hinting` isn't also disabling other forms of linting, like JS linting?

Reply:
OH Good catch! 😣 It disabled JS, too, my bad.

Reply:
Resolved by replacing `ember-cli-template-lint` module with `ember-template-lint`. The former was what added the Ember tests for template linting. The latter simply installed the linting capabilities.

Reply:
Sweet!